client/value.go

```diff
@@ -29,6 +29,10 @@ func NewFieldValue(t CType, val NormalValue) *FieldValue {
 }
 
 func (val FieldValue) Value() any {
+	jsonVal, ok := val.value.JSON()
+	if ok {
+		return jsonVal.Unwrap()
```

Comment:
Sorry for the late suggestion, I could have (but didn't) spot this earlier:

suggestion: Would it not be more correct to host this logic in `NormalValue`, and change this func to:
```
func (val FieldValue) Value() any {
  return val.value.Unwrap()
}
```
Otherwise `NormalValue`.`Unwrap` is not fully unwrapping the values.

Reply:
turns out json normal value is already doing this, so all this stuff in `FieldValue` was not necessary.
It seems like I put them there because I was confused a bit by errors due to different `int64` vs `float64` types and thought that would fix it.
Anyway thanks a lot, Andy, for helping clarify this stuff.

Reply:
Ah sweet, thanks Islam :)